tools/tensorflow_docs/api_generator/generate_lib.py

```diff
@@ -167,7 +167,7 @@ def title(self):
   @property
   def children(self):
     return sorted(
-        self._children, key=lambda x: (x.full_name.upper(), x.full_name))
+        self._children, key=lambda x: (x.full_name))
```

Comment:
The below code is consistent with Google's python linter.

```
return sorted(self._children, key=lambda x: x.full_name)
```

This sorting order is consistent with the All symbols page so LGTM